test/services/credential-service/credentials.test.js

```diff
@@ -158,13 +158,13 @@ describe('Credential service tests', () => {
     const username = 'someUser';
     const _credential = {
       secret: 'password',
-      scopes: 'someScope',
+      scopes: ['someScope'],
       someProperty: 'propVal'
     };
 
     before(() => {
       Object.assign(config.models.credentials.properties.oauth2.properties, {
-        scopes: { type: 'string' },
+        scopes: { type: 'array', items: { tyep: 'string' } },
```

Comment:
typo `tyep` , and if it works with this error I would just remove the line, since it is not needed

Reply:
Correct, this line is useless and can be removed. Thanks for spotting this.